addons/linecompass/functions/fnc_onDraw.sqf

```diff
@@ -0,0 +1,208 @@
+    _control setVariable [QGVAR(color), _color];
+    _control setVariable [QGVAR(offset), _offset];
+    _control setVariable [QGVAR(lineIndex), _lineIndex];
+
+    // Shift
+    private _otherMarkerControl = _overlapCacheLineIndices param [_lineIndex, nil];
+    if !(isNil "_otherMarkerControl") then {
+        // Compare
+        private _otherOffset = _otherMarkerControl getVariable QGVAR(offset);
+        if (abs _otherOffset < abs _offset) then {
+            // Swap
+            _offset = _otherOffset;
+            private _tmp = _otherMarkerControl;
+            _otherMarkerControl = _control;
+            _control = _tmp;
+        } else {
+            _overlapCacheLineIndices set [_lineIndex, _control];
+        };
+
+        // Direction
+        private _shiftDirection = if (_offset == 0) then {
+            1
+        } else {
+            _offset / abs _offset // 1 or -1
+        };
+
+        // Shift
+        private _shiftedLineIndex = _lineIndex;
+
+        while {!isNil "_otherMarkerControl"} do {
+            _shiftedLineIndex = _shiftedLineIndex + _shiftDirection;
+            _otherMarkerControl setVariable [QGVAR(lineIndex), _shiftedLineIndex];
+
+            private _tmp = _otherMarkerControl;
+            _otherMarkerControl = _overlapCacheLineIndices param [_shiftedLineIndex, nil];
+            _overlapCacheLineIndices set [_shiftedLineIndex, _tmp];
+        };
+    } else {
+        _overlapCacheLineIndices set [_lineIndex, _control];
+    };
+
+    _nextLineMarkerControl = _nextLineMarkerControl + 1;
+
+} forEach GVAR(lineMarkers);
+
+// Remove the unused controls
+if (_nextLineMarkerControl < count GVAR(lineMarkerControlPool)) then {
+    for "_i" from _nextLineMarkerControl to (count GVAR(lineMarkerControlPool) - 1) do {
+        ctrlDelete (GVAR(lineMarkerControlPool) deleteAt _nextLineMarkerControl);
+    };
+};
+
+private _yPos = [PY(0.6), PY(2)] select GVAR(SwapOrder);
+private _hPos = PX(2.2);
+private _wPos = PY(0.3);
+
+{
+    if (ctrlShown _x) then {
+
+        private _lineIndex = _x getVariable QGVAR(lineIndex);
+        private _color = _x getVariable QGVAR(color);
+        private _alpha = (2.5 + ((_lineIndex - floor (_viewDirection / 5)) * 5) - (_viewDirection % 5)) call FUNC(getAlphaFromX);
+
+        _x ctrlSetPosition [PX(_lineIndex * 2.5 + 0.15), _yPos, _hPos, _wPos];
+        _x ctrlSetTextColor [_color select 0, _color select 1, _color select 2, (_color select 3) * _alpha];
+        _x ctrlCommit 0;
+    };
+} forEach GVAR(lineMarkerControlPool);
+
+// Icon marker
+private _nextIconMarkerControl = 0;
+
+private _player = call CBA_fnc_currentUnit;
+
+private _yOffSet = [PY(0.75), PY(2.15)] select GVAR(SwapOrder);
+
+{
+    _x params ["_unit", "_color", "_icon", "_size"];
+
+    // Check if the unit is not the player himself and alive.
+    if (alive _unit && _unit != _player && (isNull objectParent _player || {!(_unit in crew objectParent _player)})) then {
```

Comment:
I can see this one causing confusion, but that is then a new feature to change the behaviour

Reply:
Thinking about this some more, i feel this should behave the same as DUI.

Just render the units in group + special track. Does not matter if dead or not. Depending on how respawn in the missino works, a dead unit gets either removed immediately or on respawn.